setup.py

```diff
@@ -15,6 +15,7 @@
           'python-fmask'
       ],
       dependency_links=[
-          'hg+https://bitbucket.org/chchrsc/rios/get/1.4.4.zip#egg=rios-1.4.4',
+          'hg+https://bitbucket.org/chchrsc/rios/get/rios-1.4.5.zip#egg=rios-1.4.5',
           'hg+https://bitbucket.org/chchrsc/python-fmask/get/python-fmask-0.4.5.zip#egg=python-fmask-0.4.5'
-      ])
+      ]
+      )
```

Comment:
No newline at end of file!
Boo!

All good! (provided the code is functional which I have not checked :smile: )

Reply:
fixed